packages/db/src/search/field-mapping.ts

```diff
@@ -123,6 +123,12 @@ const collectionMapping : { [key: string]: Function } = {
   updatedAt: (value?: string) => ({
     updated_at: value && new Date(Number(value)),
   }),
+  reportToEms: (value?: string) => ({
```

Comment:
are these from the api docs or from transferring from ES (or both)? just curious about where to verify this is coming from